bedrock/styleguide/templates/styleguide/identity/webmaker-branding.html

```diff
@@ -27,7 +27,7 @@
     <li>use our logos to, or in connection with, content that disparages us or sullies our reputation</li>
   </ul>
   <div class="guidelines-logo">
-    <img src="{{ static('img/styleguide/identity/webmaker/logo.png') }}" alt="Logo" />
+    {{ high_res_img('styleguide/identity/webmaker/logo.png', {'alt': 'Logo'}) }}
     <h3>Download</h3>
     <ul class="button-list">
       <li><a href="https://assets.mozilla.org/portal/assets/#asset/76" class="button-sand">EPS</a></li>
```

Comment:
Looks like we need to get the new logo into the assets site so we can update the download links. I'm not sure who to nag to get that done. Martell maybe?